platform-sdk/swirlds-common/src/testFixtures/java/module-info.java

```diff
@@ -21,6 +21,7 @@
     requires transitive com.hedera.pbj.runtime;
     requires com.swirlds.config.extensions.test.fixtures;
     requires com.swirlds.logging;
+    requires com.swirlds.merkledb;
```

Comment:
I do not think this dependency should be added. Test fixtures is generic testing utilities. Rather than adding this dependency for the `TestFileSystemManager.resetMerkleDb` method, it can be implemented in `MerkleDbTestUtils` or something similar that is specific to merkle db.

Reply:
Moved the `resetMerkleDb` method to `MerkleDbTestUtils` in [this commit](https://github.com/hashgraph/hedera-services/pull/14695/commits/98e06917eab4d3172446e580023cfb1d1569e8b7). Let me know if you had that in mind or I need to update it further.